### PR TITLE
new first line for zone API docs

### PR DESCRIPTION
**Pull request by sethladd**
I never understand what a "dynamic extent" is. I'd like to propose a more "developer friendly" definition of a Zone in the API docs.

Open to wordsmithing.

### CODE REVIEW

**Comment by sethladd** [comment]
cc @floitschG

File: sdk/lib/async/zone.dart [hunk 1]
```diff
@@ -224,10 +224,14 @@ abstract class ZoneDelegate {
...
- * the callback of a `future.then` is executed in the same zone as the one where
- * the `then` was invoked.
+ * A Zone is an execution context that persists across
+ * async functions.
+ * 
+ * Asynchronous callbacks are executed in the zone they have been queued in.
+ * For example, the callback of a `future.then` is executed in the same zone as
+ * the one where the `then` was invoked.
+ * 
+ * A Zone represents the asynchronous version of a _dynamic extent_.
```
Comment (johnmccutchan) [inline sdk/lib/async/zone.dart]: I say we just drop this "dynamic extent" line.

Comment (floitschG) [inline sdk/lib/async/zone.dart]: Fine with me.

File: sdk/lib/async/zone.dart [hunk 2]
```diff
@@ -224,10 +224,14 @@ abstract class ZoneDelegate {
 }
 
 /**
- * A Zone represents the asynchronous version of a dynamic extent. Asynchronous
- * callbacks are executed in the zone they have been queued in. For example,
- * the callback of a `future.then` is executed in the same zone as the one where
- * the `then` was invoked.
+ * A Zone is an execution context that persists across
+ * async functions.
```
Comment (lrhn) [inline sdk/lib/async/zone.dart]: Put it on one line.

File: sdk/lib/async/zone.dart [hunk 3]
```diff
@@ -224,10 +224,14 @@ abstract class ZoneDelegate {
...
+ * 
+ * Asynchronous callbacks are executed in the zone they have been queued in.
```
Comment (lrhn) [inline sdk/lib/async/zone.dart]: Not sure "queued" is the right word. Maube "registered" or "added"?
Not great either.

**Comment by lrhn** [comment]
@floitschG is working on a larger rewriting of the `Zone` docs, including the first line.

**Comment by sethladd** [comment]
Thanks for the review. Glad to hear there's new Zone API docs coming.

**Pull request closed without merging**